tests/integration/security/filebased_tls_origination/egress_gateway_origination_test.go

```diff
@@ -305,11 +307,6 @@ func setupEcho(t *testing.T, ctx resource.Context) (echo.Instance, echo.Instance
 		}).
 		BuildOrFail(t)
 
-	// Create a sidecarScope to only allow traffic to service namespace so that the traffic only goes
-	// through the egress gateway set up in service namespace. This way client cannot directly call our server
-	// in the same namespace("app") and is forced to route traffic to egress gateway
-	createSidecarScope(t, ctx, appsNamespace, serviceNamespace)
```

Comment:
Why is this removed?

Reply:
Since we are going from {client Namespace} -> {istio-system} -> {server namespace}  we don't need sidecar scope anymore. Now the server is deployed in a completely different namespace